packages/graph/hash_graph/lib/graph/src/subgraph.rs

```diff
@@ -0,0 +1,141 @@
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+use type_system::uri::VersionedUri;
+use utoipa::{openapi, ToSchema};
+
+use crate::{
+    knowledge::{EntityId, KnowledgeGraphQueryDepth, PersistedEntity, PersistedLink},
+    ontology::{
+        OntologyQueryDepth, PersistedDataType, PersistedEntityType, PersistedLinkType,
+        PersistedPropertyType,
+    },
+};
+
+// TODO - This is temporary and introduced for consistency, we need to introduce actual IDs for
```

Comment:
```suggestion
// TODO: This is temporary and introduced for consistency, we need to introduce actual IDs for
```

Reply:
We have `TODO -` all over the Rust codebase. Unless we establish one pattern (and hopefully lint on it) I think we shouldn't bother with these sorts of comments in PRs

Reply:
We use `TODO: ` at many more places than `TODO - `. I haven't noticed, we already have `TODO - `. Anyway, you used `TODO: ` and `TODO - ` in the same file. I'd be fine unifying it in the Graph Rust code base in another PR, though.